ui-components/src/config/NavigationContext.tsx

```diff
@@ -26,7 +26,11 @@ export interface NavigationContextProps {
 }
 
 export const NavigationContext = createContext<NavigationContextProps>({
-  LinkComponent: (props) => <a href={props.href}>{props.children}</a>,
+  LinkComponent: (props) => (
```

Comment:
I'll revert this change and push it up, since it looks like it's fixed in https://github.com/bloom-housing/bloom/pull/1309